bugbug/nlp.py

```diff
@@ -25,6 +48,11 @@ def spacy_token_lemmatizer(text):
 
 class SpacyVectorizer(TfidfVectorizer):
     def __init__(self, *args, **kwargs):
+
+        # Detect when the optional dependency is missing
```

Comment:
Nit: `# Detect when the spacy optional dependency is missing` (at some point we might split Spacy and Gensim into separate optional deps)